nerfstudio/utils/profiler.py

```diff
@@ -46,7 +45,7 @@
 class time_function(ContextDecorator):  # pylint: disable=invalid-name
     """Decorator/Context manager: time a function call or a block of code"""
 
-    def __init__(self, name: str):
+    def __init__(self, name):
```

Comment:
Why was the type annotation removed?

Reply:
In VSCode the "@time_function" decorator was underlined as typing error as it didn't consider the "__new__" function I guess. I can also use the signature "str | Callable" for __init__ where the "Callable" won't ever be used. I think this would also fix the VSCode errors. What do you think? 

...I really don't know why the __new__ signature is not picked up.

Reply:
Yea, I would go with that